apps/etterlatte-utbetaling/src/main/kotlin/no/nav/etterlatte/utbetaling/iverksetting/oppdrag/OppdragMapper.kt

```diff
@@ -50,6 +53,17 @@ object OppdragMapper {
                     },
                 )
 
+                if (erGRegulering) {
+                    tekst140.add(
+                        Tekst140().apply {
+                            val fraOgMed = utbetaling.utbetalingslinjer.first().periode.fra
+                            tekst = "Grunnbeløpet har økt fra 1. mai ${fraOgMed.year}. De aller fleste vil få etterbetalt i juni."
+                            datoTekstFom = fraOgMed.toXMLDate()
+                            datoTekstTom = LocalDate.of(fraOgMed.year, fraOgMed.month.plus(1), 20).toXMLDate()
```

Comment:
Er litt bekymret for om denne teksten står på alt av utbetaling (revurdering, og førstegangsinnvilgelse) fra 1.5 til 20.6.